test/unit/tracks/text-track-settings.test.js

```diff
@@ -9,6 +9,16 @@ const tracks = [{
   label: 'test'
 }];
 
+const defaultSettings = {
```

Comment:
We don't actually need to worry about the `Services` and `WebVTTSet` in vttjs because that's only used in native firefox. However, I wish that it was possible to get a build of vttjs without those. Also, we're still using a fork to enable our usage of vttjs the way we want.
I think I need to load up this PR and think about it some more.